x/ibc/applications/transfer/types/genesis.pb.go

```diff
@@ -25,16 +25,16 @@ const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
 // GenesisState defines the ibc-transfer genesis state
 type GenesisState struct {
-	PortId      string `protobuf:"bytes,1,opt,name=port_id,json=portId,proto3" json:"port_id,omitempty" yaml:"port_id"`
-	DenomTraces Traces `protobuf:"bytes,2,rep,name=denom_traces,json=denomTraces,proto3,castrepeated=Traces" json:"denom_traces" yaml:"denom_traces"`
-	Params      Params `protobuf:"bytes,3,opt,name=params,proto3" json:"params"`
+	PortId      string       `protobuf:"bytes,1,opt,name=port_id,json=portId,proto3" json:"port_id,omitempty" yaml:"port_id"`
+	DenomTraces []DenomTrace `protobuf:"bytes,2,rep,name=denom_traces,json=denomTraces,proto3" json:"denom_traces" yaml:"denom_traces"`
```

Comment:
the `castrepeated` option is not working for me. Any ideas on how to fix this @colin-axner @amaurymartiny @anilCSE?

Reply:
your tools version or binary might be off. I noticed after some recent tooling changes `make proto-all` started producing incorrect outputs for me

Reply:
might be a long shot, but maybe porting an old version of the make script and see if it works?

Reply:
The short-term fix is: https://github.com/cosmos/cosmos-sdk/pull/7410 (be sure to remove the old `protoc-gen-gocosmos` if you have one).

The long-term fix is using Docker to run `make proto-gen`